types/kv-model.d.ts

```diff
@@ -164,5 +194,8 @@ export declare class KeyValueModel extends ModelBase {
    *
    * @header KeyValueModel.iterateKeys(filter)
    */
-  iterateKeys(filter?: Filter, options?: Options): Iterator<Promise<string>>;
+  static iterateKeys(
+    filter?: Filter,
+    options?: Options,
+  ): Iterator<Promise<string>>;
```

Comment:
This is not accurate. It should be `AsyncIterator<string>` which is only available with `esnext`. See https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-3.html.

```
Caveats
Keep in mind that our support for async iterators relies on support for Symbol.asyncIterator to exist at runtime. You may need to polyfill Symbol.asyncIterator, which for simple purposes can be as simple as: (Symbol as any).asyncIterator = Symbol.asyncIterator || Symbol.for("Symbol.asyncIterator");
You also need to include esnext in your --lib option, to get the AsyncIterator declaration if you do not already have it.
Finally, if your target is ES5 or ES3, you’ll also need to set the --downlevelIterators flag.
```